src/hal/src/pso/mod.rs

```diff
@@ -145,6 +145,19 @@ impl From<Stage> for ShaderStageFlags {
     }
 }
 
+impl fmt::Display for Stage {
```

Comment:
does it make sense to have this implementation? Arguably, it brings nothing extra on top of `Debug`, which is derived

Reply:
We don't need this, I basically added it because it made the text read better (no random capitals) and because it did no harm. I'm happy to remove it and use the Debug format if you prefer?

Reply:
The text is only seen on error, it's not something you read every day anyway. In general, I like less code if possible, but we can leave that implementation since you already wrote it.